docs/custom_text_ops.md

```diff
@@ -0,0 +1,365 @@
+## Operator Schemas
+
+### Auxiliary String Operator
+
+|**Operator**|**Support State**|
+|------------|-----------------|
+|StringEqual |  Supported        |
+|StringHash  |  Supported        |
+|StringToHashBucketFast|Supported|
+|StringJoin  | Supported         |
+|StringRegexReplace| Supported  |
+|StringSplit | Supported       |
+|StringUpper  | Supported     |
+|StringSlice | In next version |
+|StringLength | In next version |
+|StringToVector|  In next version|
+|VectorToString| In next version |
+
+
+
+### Tokenizer
+
+|**Operator**|**Support State**|
+|------------|-----------------|
+|GPT2Tokenizer| Supported       |
+|BertTokenizer| In next version |
+|XLNetTokenizer| In next version |
+
+
+## Auxiliary String Operator
+
+[TODO: Add existing operators]
+
+### <a name="StringSlice"></a><a name="StringSlice">**StringSlice**</a>
+Do the slice operation to each string element in input tensor. Similar to string slice in python
+```python
```

Comment:
Shoud we parse these examples to run all of them in a unit test?

Reply:
I agree with you that those examples should be added to the unit test, but I think we do not need to parse them. The reason for parsing the doc is we are afraid of the consistency between op schema and our code. But in my point of view,  the schema should not be changed too often since it establishes.

I will add those examples into the unit test when I am implementing those processors.